test/e2e/quadlet/file-abs.build

```diff
@@ -0,0 +1,5 @@
+## assert-podman-final-args --file=/etc/containers/systemd/Containerfile
```

Comment:
Why not use just `assert-podman-args`?

Reply:
I used `final-args`, because this has `--file` as the last argument instead of a context directory.